modules/gravatar-hovercards.php

```diff
@@ -149,6 +169,40 @@ function grofiles_get_avatar( $avatar, $author ) {
 				grofiles_gravatars_to_append( $user->ID );
 		}
 	} else if ( isset( $author->comment_type ) ) {
+		if ( $is_amp ) {
+			if ( 1 === preg_match( '/avatar\/([a-zA-Z0-9]+)\?/', $avatar, $email_hash ) ) {
+				$email_hash  = $email_hash[1];
+				$cache_group = 'gravatar_profiles_';
+				$cache_key   = 'gravatar_profile_' . $email_hash;
+
+				$response_body = wp_cache_get( $cache_key, $cache_group );
+				if ( false === $response_body ) {
+					$response = wp_remote_get( 'https://en.gravatar.com/' . $email_hash . '.json' );
+
+					if ( is_array( $response ) && ! is_wp_error( $response ) ) {
+						$response_body = json_decode( $response['body'] );
+						wp_cache_set( $cache_key, $response_body, $cache_group, 60 * MINUTE_IN_SECONDS );
+					}
+				}
+
+				$profile      = $response_body->entry[0];
+				$display_name = $profile->displayName; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+				$location     = $profile->currentLocation; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+				$description  = $profile->aboutMe; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
```

Comment:
This can create some notices when info is missing from some profiles:

```
Notice: Undefined property: stdClass::$currentLocation in wp-content/plugins/jetpack-dev/modules/gravatar-hovercards.php on line 190

Notice: Undefined property: stdClass::$aboutMe in wp-content/plugins/jetpack-dev/modules/gravatar-hovercards.php on line 191
```

Reply:
@jeherve These notices should be fixed now